packages/cache/src/Testing/TestingCache.php

```diff
@@ -0,0 +1,209 @@
+<?php
+
+namespace Tempest\Cache\Testing;
+
+use Closure;
+use PHPUnit\Framework\Assert;
+use Psr\Cache\CacheItemInterface;
+use Psr\Clock\ClockInterface;
+use Stringable;
+use Symfony\Component\Cache\Adapter\ArrayAdapter;
+use Tempest\Cache\Cache;
+use Tempest\Cache\Config\CustomCacheConfig;
+use Tempest\Cache\GenericCache;
+use Tempest\Cache\GenericLock;
+use Tempest\DateTime\DateTimeInterface;
+use Tempest\DateTime\Duration;
+use Tempest\Support\Random;
+
+final class TestingCache implements Cache
```

Comment:
`TestingCache` or `TestCache`?

Reply:
Anything as long as it's consistent with the other components (eg. `TestingStorage`). I prefer `TestingCache` as in "the cache for testing".

Reply:
Fair enough!